example/example_bar_plot.m

```diff
@@ -0,0 +1,53 @@
+function example_bar_plot()
```

Comment:
Please either move the `example` folder into the `test` folder to avoid cluttering the root folder. You could also make this into a test suite (see the other functions in `test/suites`), but I think in this case that might be more work and less convenient to execute.